testcases/master/template-android.jinja2

```diff
@@ -0,0 +1,17 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% set OS_INFO = OS_INFO|default("android") %}
+{% set DEPLOY_OS = DEPLOY_OS|default("android") %}
+
+{# urls #}
+{% set DOCKER_IMAGE = DOCKER_IMAGE|default("yongqinliu/linaro-android-docker:0.1") %}
+{% set use_docker_image_for_test_target = use_docker_image_for_test_target|default(true) %}
```

Comment:
Maybe there two variables belong in projects/lkft-android/fastboot.jinja2 ?

Reply:
no, I put them here just want to make it clear that all the android jobs should be docker method, at least by default.